spec/support/factory_bot.rb

```diff
@@ -1,4 +1,6 @@
 #Factory Bot configuration
   RSpec.configure do |config|
+    include ActionDispatch::TestProcess
```

Comment:
@JuanfraM it enabled me able to upload an image on the factory.
Since now I got the default image, I don't see the use anymore, cause image uploads can be made through the model.